plugins/modules/keycloak_client_rolescope.py

```diff
@@ -0,0 +1,280 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) Ansible project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: keycloak_client_rolescope
+
+short_description: Allows administration of Keycloak client roles scope to restrict the usage of certain roles to a other specific client applications.
+
+version_added: 8.6.0
+
+description:
+    - This module allows you to add or remove Keycloak roles from clients scope via the Keycloak REST API.
+      It requires access to the REST API via OpenID Connect; the user connecting and the client being
+      used must have the requisite access rights. In a default Keycloak installation, admin-cli
+      and an admin user would work, as would a separate client definition with the scope tailored
+      to your needs and a user having the expected roles.
+
+    - Client O(client_id) must have full_scope_allowed set to V(false).
```

Comment:
You can use
```suggestion
    - Client O(client_id) must have O(ignore:full_scope_allowed) set to V(false).
```
if that is not a real option, or
```suggestion
    - Client O(client_id) must have O(community.general.keycloak_client#module:full_scope_allowed) set to V(false).
```
to reference the `full_scope_allowed` option of the `community.general.keycloak_client` module.